README.md

```diff
@@ -1,205 +1,99 @@
 <p align="center">
-  <img src="assets/logo.svg" alt="Meilisearch" width="200" height="200" />
+  <img src="assets/meilisearch-logo-light.svg?sanitize=true#gh-light-mode-only">
+  <img src="assets/meilisearch-logo-dark.svg?sanitize=true#gh-dark-mode-only">
 </p>
 
-<h1 align="center">Meilisearch</h1>
-
 <h4 align="center">
   <a href="https://www.meilisearch.com">Website</a> |
   <a href="https://roadmap.meilisearch.com/tabs/1-under-consideration">Roadmap</a> |
   <a href="https://blog.meilisearch.com">Blog</a> |
-  <a href="https://fr.linkedin.com/company/meilisearch">LinkedIn</a> |
-  <a href="https://twitter.com/meilisearch">Twitter</a> |
   <a href="https://docs.meilisearch.com">Documentation</a> |
-  <a href="https://docs.meilisearch.com/faq/">FAQ</a>
+  <a href="https://docs.meilisearch.com/faq/">FAQ</a> |
+  <a href="https://slack.meilisearch.com">Slack</a>
 </h4>
 
 <p align="center">
   <a href="https://github.com/meilisearch/meilisearch/actions"><img src="https://github.com/meilisearch/meilisearch/workflows/Cargo%20test/badge.svg" alt="Build Status"></a>
   <a href="https://deps.rs/repo/github/meilisearch/meilisearch"><img src="https://deps.rs/repo/github/meilisearch/meilisearch/status.svg" alt="Dependency status"></a>
   <a href="https://github.com/meilisearch/meilisearch/blob/main/LICENSE"><img src="https://img.shields.io/badge/license-MIT-informational" alt="License"></a>
-  <a href="https://slack.meilisearch.com"><img src="https://img.shields.io/badge/slack-meilisearch-blue.svg?logo=slack" alt="Slack"></a>
-  <a href="https://github.com/meilisearch/meilisearch/discussions" alt="Discussions"><img src="https://img.shields.io/badge/github-discussions-red" /></a>
   <a href="https://app.bors.tech/repositories/26457"><img src="https://bors.tech/images/badge_small.svg" alt="Bors enabled"></a>
 </p>
 
-<p align="center">⚡ Lightning Fast, Ultra Relevant, and Typo-Tolerant Search Engine 🔍</p>
+<p align="center">⚡ A lightning-fast search engine that fits effortlessly into your apps, websites, and workflow 🔍</p>
 
-**Meilisearch** is a powerful, fast, open-source, easy to use and deploy search engine. Both searching and indexing are highly customizable. Features such as typo-tolerance, filters, and synonyms are provided out-of-the-box.
-For more information about features go to [our documentation](https://docs.meilisearch.com/).
+Meilisearch helps you shape a delightful search experience in a snap, offering features that work out-of-the-box to speed up your workflow.
```

Comment:
This is just me being picky, but could we try to use another word instead of "great"? I feel it's so used it doesn't mean anything anymore... Some suggestions: unique, top-notch, quality, remarkable...

Reply:
I picked "great" mostly because I needed and adjective, but didn't want to be super specific. 

Edited my initial suggestion to use "top-notch".

Reply:
I've simply reused the catchphrase used on meilisearch.com as consistency "Shape a delightful search experience in a snap"

Reply:
That's a good point. Repeating that sentence could help reinforce Meilisearch's brand.

I stand by my original comments (not a fan of both "shape" and "delightful" in this context), but marketing is definitely not my top skill. I'm fine with skipping this change if we want to keep messaging consistent.

Reply:
I let the same "brand" phrase as the landing for now, something I will ensure we are ok with later with Ferdi & Quentin 😉